src/test/java/org/sagebionetworks/web/client/FeatureFlagConfigTest.java

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.web.client;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import com.google.gwt.json.client.JSONBoolean;
+import com.google.gwt.json.client.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.sagebionetworks.web.client.cookie.CookieProvider;
+
+public class FeatureFlagConfigTest {
+
+  @Mock
+  CookieProvider mockCookieProvider;
+
+  @Mock
+  JSONObject mockJsonObject;
+
+  private FeatureFlagConfig featureFlagConfig;
+  private static final String FEATURE_NAME = "testFeature";
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  public void testFeatureEnabled() {
+    when(mockJsonObject.get(FEATURE_NAME))
+      .thenReturn(JSONBoolean.getInstance(true));
+    featureFlagConfig =
+      new FeatureFlagConfig(mockJsonObject, mockCookieProvider);
+
+    assertTrue(featureFlagConfig.isFeatureEnabled(FEATURE_NAME));
+  }
+
+  @Test
+  public void testFeatureDisabled() {
+    when(mockJsonObject.get(FEATURE_NAME))
+      .thenReturn(JSONBoolean.getInstance(false));
+    featureFlagConfig =
+      new FeatureFlagConfig(mockJsonObject, mockCookieProvider);
+
+    assertFalse(featureFlagConfig.isFeatureEnabled(FEATURE_NAME));
+  }
+}
```

Comment:
Need at least 2 more tests:
* Verify that `true` is returned if the feature is disabled but experimental mode is enabled
* Verify that the experimental mode value is returned if an exception is thrown when `mockJsonObject.get(FEATURE_NAME)` is called